### PR TITLE
Add package flac.0.1.6

### PR DESCRIPTION
**Pull request by toots**
This is a bugfix release, fixing a memory leak when decoding flac data.

### CODE REVIEW

File: packages/flac/flac.0.1.6/opam [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  ["libflac-dev"] {os-distribution = "debian"}
+  ["libflac-dev"] {os-distribution = "ubuntu"}
+  ["flac"] {os-distribution = "nixos"}
+  ["flac"] {os = "macos" & os-distribution = "homebrew"}
+]
+bug-reports: "https://github.com/savonet/ocaml-flac/issues"
+dev-repo: "git+https://github.com/savonet/ocaml-flac.git"
+synopsis:
+  "Interface for the Free Lossless Audio Codec otherwise known as FLAC"
+flags: light-uninstall
```
Comment (toots) [inline packages/flac/flac.0.1.6/opam:42]:
```suggestion
```

File: packages/flac/flac.0.1.6/opam [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "OCAMLFLAGS=-ccopt -I/usr/local/include -cclib -L/usr/local/lib"
+    "--prefix"
+    prefix
+  ] {os = "macos"}
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["ocamlfind" "remove" "flac"]
```
Comment (toots) [inline packages/flac/flac.0.1.6/opam:21]:
```suggestion
```

File: packages/flac/flac.0.1.6/opam [hunk 3]
```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-flac"
+build: [
+  ["./bootstrap"]
```
Comment (toots) [inline packages/flac/flac.0.1.6/opam:6]:
```suggestion
```

File: packages/flac/flac.0.1.6/opam [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  "ogg"
+]
+depexts: [
+  ["flac-dev"] {os-distribution = "alpine"}
+  ["flac"] {os-distribution = "archlinux"}
+  ["flac-devel"] {os-distribution = "centos"}
+  ["flac-devel"] {os-distribution = "fedora"}
+  ["flac-devel"] {os-family = "suse"}
+  ["libflac-dev"] {os-distribution = "debian"}
+  ["libflac-dev"] {os-distribution = "ubuntu"}
```
Comment (toots) [inline packages/flac/flac.0.1.6/opam:32]:
```suggestion
  ["libflac-dev"] {os-family = "debian"}
```

File: packages/flac/flac.0.1.6/opam [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    "--prefix"
+    prefix
+  ] {os = "macos"}
+  [make]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml" {>= "4.02"}
```
Comment (toots) [inline packages/flac/flac.0.1.6/opam:21]:
```suggestion
  "ocaml" {>= "4.03"}
```

**Comment by kit-ty-kate** [comment]
I had a brief look at your code and I couldn't find any runtime use of `ocamlfind`. Also the previous version had the `build` flag so I felt like it's safe to change it. If this is correct could you return the fix upstream? If it is not feel free to remove my commit, sorry in advance.

**Comment by kit-ty-kate** [comment]
thanks!

**Comment by toots** [comment]
Thks for the fix. It has been backported!

**Pull request merged**